contacts.js

```diff
@@ -10,6 +10,10 @@ var submitContacts = () => {
 		var html = contactItem(name, phone);
 
 		$('#contact-list').append(html);
+		$('#contact-name').val('');
```

Comment:
You already have this `$('#contact-name').val('');` inside a variable in line 8 and 9.

You can do `name.val('')` and `phone.val('')` instead